packages/rocketchat-livechat/server/unclosedLivechats.js

```diff
@@ -79,6 +79,7 @@ RocketChat.settings.get('Livechat_agent_leave_action', function(key, value) {
 });
 
 UserPresenceMonitor.onSetUserStatus((user, status/* , statusConnection*/) => {
+	RocketChat.Livechat.notifyAgentStatusChanged(user._id, status);
```

Comment:
@MarcosSpessatto, I want to suggest creating a new file for this implementation. Reviewing this PR I realized that it would be better having a new `UserPresenceMonitor.onSetUserStatus` callback because all of the implementation here is related to `unclosed Livechats` so, IMO we could create a specific file for the original purpose. 

What do you think?